pkg/util/validation/limits_test.go

```diff
@@ -378,7 +380,7 @@ testuser:
 
 	differentUserOverride := `
 differentuser:
-  alertmanager_notification_limits_per_integration:
+  alertmanager_notification_rate_limit_per_integration:
```

Comment:
nice catch

Reply:
It took me a while to understand why the test was failing now :facepalm:. This test didn't do strict unmarshaling previously, so it wasn't really testing anything I guess.